### ISSUE TITLE
Solving a problem with psql

### ISSUE THREAD

**Issue by TuckerJD** [SpeciesFileGroup/install_taxonworks#8]
I don't normally use psql (PostgreSQL command line interface) but I had an occasion to invoke it recently. I got an error telling me that it could not find `/usr/local/opt/readline/lib/libreadline.6.dylib`. I traced it to the fact that I had two versions of `readline` in my Cellar, and `/usr/local/opt/readline/` was linked to  readline/7.0.3_1 rather than readline/6.3.8.
I user this StackOverflow item (https://stackoverflow.com/questions/21488778/cant-run-psql-command-keep-getting-the-same-error) to help me both identify and fix the problem.
Basically, I told brew to use the older 6.3.8 version of readline with `brew switch readline 6.3.8`

I could have re-installed PostgreSQL, but since I have active databases, and upgrading PostgreSQL has proven problematic in the past, I took the simpler path of using an earlier version of readline. That solved my immediate problem, but I will be keeping an eye out for other issues related to readline.

**Comment by mjy** [comment]
Closing this, it's seems to be general psql help.
